config/credential.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
     clientID: "192695093229-kf95jtdv1jge25d86ak99e68dkrpu33o.apps.googleusercontent.com",
     clientSecret: "rXvfxBahDlJF1ORk7y660XM8"
   },
-  database: 'YOUR MONGODB URL',
+  database: 'mongodb://localhost:27017/donut1',

```

Comment:
remove this.

Reply:
@vaibhavdaren I added that deliberately since we have added instructions in readme.md to run script for setting up datatbase locally. If removed, each user will have to add the same in their credential.js file. Please provide appropriate reason for removing it, I'll be happy to do it.